tests/e2e/tests/web/regression/browser/list-key.e2e.ts

```diff
@@ -12,34 +15,90 @@ const databaseHelper = new DatabaseHelper();
 const databaseAPIRequests = new DatabaseAPIRequests();
 const apiKeyRequests = new APIKeyRequests();
 
-const dbParameters = { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port };
+const dbParameters = { host: ossStandaloneV8Config.host, port: ossStandaloneV8Config.port };
 const keyName = `TestListKey-${ Common.generateWord(10) }`;
 const elementForSearch = `SearchField-${ Common.generateWord(5) }`;
 const keyToAddParameters = { elementsCount: 500000, keyName, elementStartWith: 'listElement' };
 
 fixture `List Key verification`
-    .meta({ type: 'regression' })
+    .meta({ type: 'regression', rte: rte.standalone })
     .page(commonUrl)
     .beforeEach(async() => {
-        await databaseHelper.acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig);
-        await browserPage.addListKey(keyName, '2147476121', 'testElement');
+        await databaseHelper.acceptLicenseTermsAndAddDatabaseApi(ossStandaloneV8Config);
+        await browserPage.addListKey(keyName, '2147476121', ['testElement']);
     })
     .afterEach(async() => {
-        // Clear and delete database
-        await apiKeyRequests.deleteKeyByNameApi(keyName, ossStandaloneConfig.databaseName);
-        await databaseAPIRequests.deleteStandaloneDatabaseApi(ossStandaloneConfig);
+        await apiKeyRequests.deleteKeyByNameApi(keyName, ossStandaloneV8Config.databaseName);
+        await databaseAPIRequests.deleteStandaloneDatabaseApi(ossStandaloneV8Config);
     });
 test
-    .meta({ rte: rte.standalone })('Verify that user can search per exact element index in List key in DB with 1 million of fields', async t => {
+   ('Verify that user can search per exact element index in List key in DB with 1 million of fields', async t => {
         // Add 1000000 elements to the list key
         await populateListWithElements(dbParameters.host, dbParameters.port, keyToAddParameters);
         await populateListWithElements(dbParameters.host, dbParameters.port, keyToAddParameters);
         // Add custom element to the list key
         await browserPage.openKeyDetails(keyName);
-        await browserPage.addElementToList(elementForSearch);
+        await browserPage.addElementToList([elementForSearch]);
         // Search by element index
         await browserPage.searchByTheValueInKeyDetails('1000001');
         // Check the search result
         const result = await browserPage.listElementsList.nth(0).textContent;
         await t.expect(result).eql(elementForSearch, 'List element not found');
     });
+
+// TODO unskip when the bug is fixed https://redislabs.atlassian.net/browse/RI-6188
+test.skip
```

Comment:
no need to skip this test, because issue will be fixed in scope of this story in its branch
in order to not forget to unskip it